fastlane/Fastfile

```diff
@@ -13,6 +13,20 @@ ENV["DOWNLOAD_METADATA"]="./fastlane/download_metadata.swift"
 
 USER_ENV_FILE_PATH = File.join(Dir.home, '.simplenotemacos-env.default')
 
+workspace = 'Simplenote.xcworkspace'
+internal_scheme = 'Simplenote'
+app_store_scheme = 'Simplenote-AppStore'
+build_folder = 'build'
+
+# On CI, it's useful to skip code signing unless necessary to avoid spending
+# time syncing the certificates and profiles with match. And while not
+# neccessary, it doesn't hurt on a dev machine either.
+skip_code_signing_xcargs = [
+  'COMPILER_INDEX_STORE_ENABLE=NO',
+  "CODE_SIGN_IDENTITY=''",
+  'CODE_SIGNING_REQUIRED=NO',
+]
```

Comment:
I made it possible to pass a `Hash` to `xcargs` a while ago (https://github.com/fastlane/fastlane/pull/8051) so that it shell-escapes everything for you and makes it more readable, we should use this 😇 

```suggestion
skip_code_signing_xcargs = {
  COMPILER_INDEX_STORE_ENABLE: 'NO',
  CODE_SIGN_IDENTITY: '',
  CODE_SIGNING_REQUIRED: 'NO',
}
```